### PR TITLE
Display statistic on report

### PR DESCRIPTION
**Pull request by Hatsukikeika**
Please set the title of your pull request in the following format: `[Project id/Independent study/Issue fix/...]. project name`, eg., `E904. Integrating courses and assignments on Expertiza with Moodle`

We suggest you including the below information in your pull request:
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes.
---
About the Expertiza Bot
- If you have any questions about comments given by the expertiza-bot [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762412918), you could create a new comment in your pull request with the content `/dispute [UUID1] [UUID2]` [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762430057), then the professor and TAs will be notified.
- [Here is a video](https://tinyurl.com/internet-bots) about how to communicate to the expertiza-bot.

### CODE REVIEW

File: app/views/reports/_review_report.html.erb [hunk 1]
```diff
@@ -64,7 +64,7 @@
         <% if reviewer.id != -1 %>
           <% user = Participant.find(reviewer.id).user %>
             <!--reviewer-->
-			
+
```
Comment (boergerj) [inline app/views/reports/_review_report.html.erb:68]: Remove whitespace change

File: app/views/reports/_review_report.html.erb [hunk 2]
```diff
@@ -117,7 +119,7 @@
                 <%= render partial: 'team_score_score_awarded', locals: {bgcolor: @bgcolor, team_id: @team.id, reviewer_id: reviewer.id} %>
               <% end %>
             <% end %>
-          </td>         
+          </td>
```
Comment (boergerj) [inline app/views/reports/_review_report.html.erb:123]: Remove whitespace change

File: app/views/reports/_review_report.html.erb [hunk 3]
```diff
@@ -258,7 +275,112 @@
 		row.push("EmailID");
 		row.push("Grade");
 		row.push("Comment");
-		
+
```
Comment (boergerj) [inline app/views/reports/_review_report.html.erb:279]: Remove whitespace change

File: db/schema.rb [hunk 4]
```diff
@@ -11,10 +11,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
```
Comment (boergerj) [inline db/schema.rb:14]: Remove whitespace change

File: db/schema.rb [hunk 5]
```diff
@@ -11,10 +11,8 @@
...
 ActiveRecord::Schema.define(version: 20201125202200) do
 
-
```
Comment (boergerj) [inline db/schema.rb:17]: Remove whitespace change

File: db/schema.rb [hunk 6]
```diff
@@ -643,11 +647,12 @@
...
 
   create_table "suggestion_comments", force: :cascade do |t|
-    t.text     "comments",      limit: 65535
-    t.string   "commenter",     limit: 255
-    t.string   "vote",          limit: 255
-    t.integer  "suggestion_id", limit: 4
+    t.text     "comments",           limit: 65535
+    t.string   "commenter",          limit: 255
+    t.string   "vote",               limit: 255
+    t.integer  "suggestion_id",      limit: 4
```
Comment (boergerj) [inline db/schema.rb:663]: Remove whitespace change

File: db/schema.rb [hunk 7]
```diff
@@ -643,11 +647,12 @@
...
     t.datetime "created_at"
+    t.boolean  "visible_to_student",               default: false
```
Comment (boergerj) [inline db/schema.rb:665]: Why was this added?

File: db/schema.rb [hunk 8]
```diff
@@ -731,6 +736,7 @@
     t.integer "directory_num",              limit: 4
     t.integer "grade_for_submission",       limit: 4
     t.text    "comment_for_submission",     limit: 65535
+    t.boolean "make_public",                              default: false
```
Comment (boergerj) [inline db/schema.rb:749]: Why was this added?

File: db/schema.rb [hunk 9]
```diff
@@ -742,14 +748,14 @@
   add_index "teams_users", ["user_id"], name: "fk_teams_users", using: :btree
 
   create_table "track_notifications", force: :cascade do |t|
-    t.integer  "notification_id", limit: 4
     t.integer  "user_id",         limit: 4
-    t.datetime "created_at",                null: false
-    t.datetime "updated_at",                null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "notification_id", limit: 4, null: false
```
Comment (boergerj) [inline db/schema.rb:764]: Why are these changing?

File: db/schema.rb [hunk 10]
```diff
@@ -742,14 +748,14 @@
...
   end
 
-  add_index "track_notifications", ["notification_id"], name: "index_track_notifications_on_notification_id", using: :btree
-  add_index "track_notifications", ["user_id"], name: "index_track_notifications_on_user_id", using: :btree
+  add_index "track_notifications", ["notification_id"], name: "notification_id", using: :btree
+  add_index "track_notifications", ["user_id"], name: "user_id", using: :btree
```
Comment (boergerj) [inline db/schema.rb:768]: Why are these changing?

File: db/schema.rb [hunk 11]
```diff
@@ -812,6 +818,7 @@
   add_foreign_key "assignment_badges", "badges"
   add_foreign_key "assignment_questionnaires", "assignments", name: "fk_aq_assignments_id"
   add_foreign_key "assignment_questionnaires", "questionnaires", name: "fk_aq_questionnaire_id"
+  add_foreign_key "assignments", "assignments", column: "sample_assignment_id"
```
Comment (boergerj) [inline db/schema.rb:831]: Why is this changing?

File: db/schema.rb [hunk 12]
```diff
@@ -845,6 +852,4 @@
   add_foreign_key "tag_prompt_deployments", "tag_prompts"
   add_foreign_key "teams_users", "teams", name: "fk_users_teams"
   add_foreign_key "teams_users", "users", name: "fk_teams_users"
-  add_foreign_key "track_notifications", "notifications"
-  add_foreign_key "track_notifications", "users"
```
Comment (boergerj) [inline db/schema.rb:859]: Why are these changing?

**Review by boergerj — APPROVED** [review]

**Review by AvlWx2014 — APPROVED** [review]
